localization/de_DE.inc

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$labels = array();
+$labels['custom_from_off'] = 'Identität auswählen';
+$labels['custom_from_off_hint'] = 'Absender aus Liste auswählen';
+$labels['custom_from_on'] = 'Individuelle Identität';
```

Comment:
Is `Individuelle` really a word in German? I can't read German, but a quick check on https://www.wordreference.com/deen/Individuelle didn't lead to any result 🤔

Reply:
You're making me have some second thoughts here, because I'm now wondering whether it's maybe a little *too* literal. Hm… If I take the French texts as the original to possibly avoid a double translation (you to English and then me to German), maybe `Frei wählbare Identität` might fit things (`Identité libre`) a little better?
Though it does mean that the "button" text isn't much shorter than the hint text, but then again German does suffer from being somewhat more verbose, so that's the way it is I guess.

That aside, `Individuelle` certainly is a word, though – it's the female declension of `individuell`, as `Identität` is female in German.